packages/xdl/static/loading-page/index.html

```diff
@@ -0,0 +1,244 @@
+          </defs>
+          </svg>          
+      </div>
+      <div class="info-box-details">
+        <p class="info-box-app-name">Form Duo</p>
+        <div class="info-box-details-record">
+          <p>Runtime version</p>
+          <p>1.0.0</p>
+        </div>
+        <div class="info-box-details-record">
+          <p>Branch</p>
+          <p>main</p>
+        </div>
+        <div class="info-box-details-record">
+          <p>Path</p>
+          <p>/path/path/path</p>
+        </div>
+        <div class="info-box-details-record">
+          <p>Last commit</p>
+          <p>Jul 6, 2021, 4:51PM PT</p>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div class="bottom-bar">
```

Comment:
I noticed this bottom bar can cover the other page content and if the viewport is not tall enough, it's not possible to scroll down to see all the content.